config/Dockerfiles/centos7/Dockerfile

```diff
@@ -3,25 +3,27 @@ LABEL maintainer "https://github.com/CentOS-PaaS-SIG/linchpin"
 LABEL description "This container will verify linchpin works under Centos7"
 
 ENV HOME=/root
+ENV LC_ALL=en_US.utf8
+ENV LANG=en_US.utf8
```

Comment:
@14rcole For some reason it still fails. 
I faced the same error while setting up GitHub actions 
however i was able to fix it by setting variables as follows in shell script
```
VERSION_ID=$(cat /etc/*release | grep ^VERSION_ID | tr -d 'VERSION_ID="')

echo $VERSION_ID

if [ $VERSION_ID = "7" ]
then
    echo "This is centos7";
    export LC_ALL="en_US";
    export LANG="en_US";
fi
```
https://github.com/CentOS-PaaS-SIG/linchpin/blob/develop/.github/runmocktests.sh#L13